app/models/tokite/hook.rb

```diff
@@ -32,6 +32,7 @@ def fire!
         if payloads.none? {|payload| payload[:channel] == rule.channel && payload[:emoji] == emoji && payload[:additional_text] == additional_text }
           payloads << {
             channel: rule.channel,
+            username: (rule.display_name == "") ? "tokite" : rule.display_name,
```

Comment:
(Either way is OK) You can write `rule.display_name.presence || "tokite"` here.
http://api.rubyonrails.org/classes/Object.html#method-i-presence